openhcl/openhcl_boot/src/sidecar.rs

```diff
@@ -105,6 +105,15 @@ pub fn start_sidecar<'a>(
         }
     }
 
+    #[cfg(target_arch = "x86_64")]
+    if safe_intrinsics::cpuid(x86defs::cpuid::CpuidFunction::VersionAndFeatures.0, 0).ecx
+        & (1 << 21) == 0
```

Comment:
Use `VersionAndFeaturesEcx` to access the bit, for better documentation of what's happening here.